features/org.wso2.carbon.healthcheck.server.feature/resources/health-check-config.xml

```diff
@@ -29,6 +29,8 @@
             <HealthChecker name="SuperTenantUSHealthChecker" orderId="98" enable="true">
                 <!--<Property name="monitored.user.stores">primary,sec</Property>-->
             </HealthChecker>
+            <HealthChecker name="OOMHealthChecker" orderId="99" enable="true">
```

Comment:
I prefer this being disabled by default so that people who need this can enable.